README.md

```diff
@@ -7,7 +7,7 @@
 </p>
 <p align="center">
   <a href="https://developers.google.com/google-ads/api/docs/release-notes">
-    <img src="https://img.shields.io/badge/google%20ads-v7-009688.svg?style=flat-square">
```

Comment:
Is this rendering strangely for anyone else?

<img width="272" alt="Screenshot 2021-07-20 at 16 35 00" src="https://user-images.githubusercontent.com/19176942/126352934-501e58a6-6614-4aea-a656-5bed6e10c034.png">

Reply:
This is what it looks like for me:
![image](https://user-images.githubusercontent.com/1063061/126358130-e4430e9c-94a3-484c-b983-ed8c94eb720b.png)

Reply:
My mistake, that's git showing the difference between versions.